Apps/AdminWebClient/src/appsettings.Development.json

```diff
@@ -44,5 +44,13 @@
                 "Password": "*****"
             }
         }
+    },
+    "DocumentStorage": {
+        "SftpHostname": "ftpsvcs.hlth.gov.bc.ca",
+        "SftpPort": 22,
+        "SftpUsername": "***",
+        "SftpFolderPath": "/HGToHIBC/COVID19VacRep/DEV/ToMaximus",
+        "PrivateKeyPath": "/usr/local/HealthGateway/HG_SFTP_2021-Dev.pem",
```

Comment:
Apparently the path name has to be unique when mounting a ConfigMap so you need to change HealthGateway to sftp